psi4/src/psi4/libfock/jk.h

```diff
@@ -344,6 +348,10 @@ class PSI_API JK {
     size_t memory_overhead() const;
     /// Zero out all J, K, and wK matrices
     void zero();
+    /**
+    * Return number of shells computed during the JK build process.
+    */
+    virtual size_t computed_shells();
```

Comment:
I think `num_computed_shells` would be slightly clearer. (Yes, I can figure out this is a number when I see the return type, but when `auto` is in play, I can't.)

Reply:
I agree now that you mention it. Done!